recipes/busco/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     - sepp >=4.3.10
     - metaeuk
     - pandas
-    - bbmap
+    - bbmap =37.62 
```

Comment:
Is there some upstream docs on this (which may guide bioconda contributors for future updates)? 

Wondering if less strict pinning may also be possible, e.g. `37.*`

Reply:
Just tested with `bbmap=37.*` and it works, so I can relax that